cosmos/dbt/graph.py

```diff
@@ -487,6 +492,10 @@ def load_via_dbt_ls_without_cache(self) -> None:
                 env[DBT_TARGET_PATH_ENVVAR] = str(self.target_dir)
 
                 if self.render_config.dbt_deps and has_non_empty_dependencies_file(self.project_path):
+                    if is_cache_package_lockfile_enabled(project_path):
+                        latest_package_lockfile = _get_latest_cached_package_lockfile(project_path)
+                        if latest_package_lockfile:
+                            _copy_cached_package_lockfile_to_project(latest_package_lockfile, Path(tmpdir))
```

Comment:
```suggestion
                            _copy_cached_package_lockfile_to_project(latest_package_lockfile, tmpdir_path))
```
We already have a `tmpdir_path` as a `Path` instance for the `tmpdir`